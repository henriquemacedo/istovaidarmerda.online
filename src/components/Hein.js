import React from "react";
import styled from "styled-components";

const Wrapper = styled.section`
  display: grid;
  grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
  grid-gap: 100px;
  padding: 100px;
`;

const Hein = () => (
  <Wrapper id="hein">
    <div>
      <h2>O quê?</h2>
      <p>
        O <i>Isto Vai Dar Merda</i> é um evento que materializa a máxima “quem avisa, amigo é”, pensado para abordar
        temas que estão no “tipping point” de dar merda mas que ainda podem ser salvos de tão triste fado.
      </p>
      <p>
        O <i>Isto Vai Dar Merda</i> quer ir directo ao assunto – sem rodeios, sem paninhos quentes, sem merdas.
      </p>
      <p>
        Na sua primeira edição, realizada exclusivamente no Twitter, o tema é <i>Isto Vai Dar Merda</i> – na Pandemia.
        São 8 temas em discussão por 8 painéis de convidados – desde profissionais de saúde a economistas, músicos e
        políticos.
      </p>
      <p>A Pandemia vista por dentro e por fora – sem lentes cor-de-rosa.</p>
    </div>
    <div>
      <h2>Porquê?</h2>
      <p>
        Faz falta um espaço em que se possa falar sobre assuntos no que estão a dar de errado, pavimentando o caminho
        para salvaguardar que dêem, de facto, merda.
      </p>
      <p>
        Faz falta um espaço onde se possa fazer barulho, onde as opiniões não serão suavizadas, onde os conselhos e
        avisos não serão contritos ou dados com um sorriso para não ofender.
      </p>
      <p>Faz falta permitir que a crueza e realidade do que têm para dizer seja não apenas aceite, como celebrada.</p>
      <p>
        Faz falta um espaço onde o desconforto é visto como útil, onde a verdade venha directamente da boca de quem a
        vive – sem rodeios, sem paninhos quentes, sem merdas.
      </p>
      <p>
        O <i>Isto Vai Dar Merda</i> – na Pandemia pretende amplificar vozes que merecem mais atenção, partilhar
        perspectivas que têm de ser abordadas, e repor verdades que nunca deviam ter sido sequer questionadas.
      </p>
    </div>
  </Wrapper>
);

export default Hein;
