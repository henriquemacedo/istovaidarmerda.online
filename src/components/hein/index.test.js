import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import MockIntersectionObserver from "../../../utils/mocks/mockIntersectionObserver";
import Hein from "./index";

describe("components/hein", () => {
  beforeEach(() => {
    window.IntersectionObserver = MockIntersectionObserver;
  });

  describe("render", () => {
    it("should render without errors", () => {
      const component = render(<Hein />);

      expect(component).toMatchSnapshot();
    });
  });
});
