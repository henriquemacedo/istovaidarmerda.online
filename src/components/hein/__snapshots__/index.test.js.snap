// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/hein render should render without errors 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="styles__Wrapper-sc-1ahj33d-0 chSgbV hein"
        id="hein"
      >
        <div
          class="hein__content"
        >
          <div
            class="hein__column"
          >
            <h2>
              O quê?
            </h2>
            <p>
              O 
              <i>
                Isto Vai Dar Merda
              </i>
               é um evento que materializa a máxima “quem avisa, amigo é”, pensado para abordar temas que estão no “tipping point” de dar merda mas que ainda podem ser salvos de tão triste fado.
            </p>
            <p>
              O 
              <i>
                Isto Vai Dar Merda
              </i>
               quer ir directo ao assunto – sem rodeios, sem paninhos quentes, sem merdas.
            </p>
            <p>
              Na sua primeira edição, realizada exclusivamente no Twitter, o tema é 
              <i>
                Isto Vai Dar Merda
              </i>
               – na Pandemia. São 8 temas em discussão por 8 painéis de convidados – desde profissionais de saúde a economistas, músicos e políticos.
            </p>
            <p>
              A Pandemia vista por dentro e por fora – sem lentes cor-de-rosa.
            </p>
          </div>
          <div
            class="hein__column"
          >
            <h2>
              Porquê?
            </h2>
            <p>
              Faz falta um espaço em que se possa falar sobre assuntos no que estão a dar de errado, pavimentando o caminho para salvaguardar que dêem, de facto, merda.
            </p>
            <p>
              Faz falta um espaço onde se possa fazer barulho, onde as opiniões não serão suavizadas, onde os conselhos e avisos não serão contritos ou dados com um sorriso para não ofender.
            </p>
            <p>
              Faz falta permitir que a crueza e realidade do que têm para dizer seja não apenas aceite, como celebrada.
            </p>
            <p>
              Faz falta um espaço onde o desconforto é visto como útil, onde a verdade venha directamente da boca de quem a vive – sem rodeios, sem paninhos quentes, sem merdas.
            </p>
            <p>
              O 
              <i>
                Isto Vai Dar Merda
              </i>
               – na Pandemia pretende amplificar vozes que merecem mais atenção, partilhar perspectivas que têm de ser abordadas, e repor verdades que nunca deviam ter sido sequer questionadas.
            </p>
          </div>
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="styles__Wrapper-sc-1ahj33d-0 chSgbV hein"
      id="hein"
    >
      <div
        class="hein__content"
      >
        <div
          class="hein__column"
        >
          <h2>
            O quê?
          </h2>
          <p>
            O 
            <i>
              Isto Vai Dar Merda
            </i>
             é um evento que materializa a máxima “quem avisa, amigo é”, pensado para abordar temas que estão no “tipping point” de dar merda mas que ainda podem ser salvos de tão triste fado.
          </p>
          <p>
            O 
            <i>
              Isto Vai Dar Merda
            </i>
             quer ir directo ao assunto – sem rodeios, sem paninhos quentes, sem merdas.
          </p>
          <p>
            Na sua primeira edição, realizada exclusivamente no Twitter, o tema é 
            <i>
              Isto Vai Dar Merda
            </i>
             – na Pandemia. São 8 temas em discussão por 8 painéis de convidados – desde profissionais de saúde a economistas, músicos e políticos.
          </p>
          <p>
            A Pandemia vista por dentro e por fora – sem lentes cor-de-rosa.
          </p>
        </div>
        <div
          class="hein__column"
        >
          <h2>
            Porquê?
          </h2>
          <p>
            Faz falta um espaço em que se possa falar sobre assuntos no que estão a dar de errado, pavimentando o caminho para salvaguardar que dêem, de facto, merda.
          </p>
          <p>
            Faz falta um espaço onde se possa fazer barulho, onde as opiniões não serão suavizadas, onde os conselhos e avisos não serão contritos ou dados com um sorriso para não ofender.
          </p>
          <p>
            Faz falta permitir que a crueza e realidade do que têm para dizer seja não apenas aceite, como celebrada.
          </p>
          <p>
            Faz falta um espaço onde o desconforto é visto como útil, onde a verdade venha directamente da boca de quem a vive – sem rodeios, sem paninhos quentes, sem merdas.
          </p>
          <p>
            O 
            <i>
              Isto Vai Dar Merda
            </i>
             – na Pandemia pretende amplificar vozes que merecem mais atenção, partilhar perspectivas que têm de ser abordadas, e repor verdades que nunca deviam ter sido sequer questionadas.
          </p>
        </div>
      </div>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
